src/exported_files/text_styles.pr

```diff
@@ -38,6 +46,10 @@ class AppTextStyles {
     fontSize: {[ inject "export_measure_value" context typographyToken.value.fontSize /]},
     decoration: TextDecoration.{{ decoration }},
     letterSpacing: {{ letterSpacing.toString() }},
+    {[ if lineHeight != "null" ]}
+    height: {{ lineHeight.toString() }},
+    {[/]}
+    textStyle: const TextStyle(leadingDistribution: TextLeadingDistribution.even),
```

Comment:
@JiriTrecak So this `textStyle` property is being added but is not supported by `TextStyle`. A nested TextStyle in a TextStyle doesn't work. Therefore this breaks the importer such that it requires manual input to remove `textStyle` and replace it with `leadingDistribution: TextLeadingDistribution.even`. 

This is the error message:
```
	"message": "The named parameter 'textStyle' isn't defined. Try correcting the name to an existing named parameter's name, or defining a named parameter with the name 'textStyle'.",
```